### DISCUSSION TITLE
electron-vite  运行打包命令build:win 安装运行后报错 提示Cannot find module sqlite3

### DISCUSSION THREAD

**Discussion by FengyangMK** [alex8088/electron-vite#488] [Q&A]
<img width="890" alt="Snipaste_2024-04-25_21-35-18" src="https://github.com/alex8088/electron-vite/assets/62201738/90f75b06-1854-4dfb-ac2b-405e25cfbecf">
![builder](https://github.com/alex8088/electron-vite/assets/62201738/af8a87e4-9d71-416b-bab2-751f3269bfd8)
![vite](https://github.com/alex8088/electron-vite/assets/62201738/40b62b40-e780-4c7b-a66b-16f2fa717f32)

**Comment by alex8088** [comment]
https://cn.electron-vite.org/guide/troubleshooting#a-javascript-error-occurred-in-the-main-process-error-cannot-find-module-xxx
